src/components/card/CardSection.tsx

```diff
@@ -44,7 +44,7 @@ export type CardSectionProps = ViewProps & {
   /**
    * Other image props that will be passed to the image
    */
-  imageProps?: ImageProps;
+  imageProps?: Partial<ImageProps>;
```

Comment:
This: `imageProps?: Omit<ImageProps, 'source' | 'style'>;` feels a bit more accurate, WDYT?

Reply:
I know, but I'm not sure it's intentional, approving